iree/hal/vulkan/native_timeline_semaphore.cc

```diff
@@ -75,8 +75,9 @@ Status NativeTimelineSemaphore::Signal(uint64_t value) {
   signal_info.pNext = nullptr;
   signal_info.semaphore = handle_;
   signal_info.value = value;
-  return VkResultToStatus(logical_device_->syms()->vkSignalSemaphore(
-      *logical_device_, &signal_info));
+  return VkResultToStatus(__FILE__, __LINE__,
+                          logical_device_->syms()->vkSignalSemaphore(
+                              *logical_device_, &signal_info));
```

Comment:
Can some magic be added that simplifies the `__FILE__`, `__LINE__` passing to `VkResultToStatus`?